safe-global/web-core
# [Create Safe] feat: Status view redesign

## What it solves

Resolves #994 

## How this PR fixes it

- New design for the status view that includes a stepper
- Animated loading spinner
- Manual navigation to the safe after creation
- A new Dialog with first step hints after creation

## How to test it

1. Open `/new-safe/create`
2. Submit the form
4. Observe the new design

## ToDos

- [x] Add loading spinner with animations
- [x] Add modal on dashboard after safe creation
- [ ] Review Dialog content and status messages cc @johannesmoormann 

## Screenshots
<img width="804" alt="Screenshot 2022-11-02 at 17 21 07" src="https://user-images.githubusercontent.com/5880855/199544510-e745c56b-ffa1-43f0-9a17-9c2727c34ba7.png">
<img width="806" alt="Screenshot 2022-11-02 at 17 21 21" src="https://user-images.githubusercontent.com/5880855/199544558-5c8b2554-dd0e-43c8-8cb7-3342fbf4a5c1.png">
<img width="799" alt="Screenshot 2022-11-02 at 17 21 37" src="https://user-images.githubusercontent.com/5880855/199544604-a3589e3e-e942-4ff9-8df6-e1bd40b43e36.png">
<img width="790" alt="Screenshot 2022-11-02 at 17 21 59" src="https://user-images.githubusercontent.com/5880855/199544684-a239f7e9-311e-4e25-a1a4-5d05b6ab5cf2.png">
<img width="1119" alt="Screenshot 2022-11-02 at 17 20 15" src="https://user-images.githubusercontent.com/5880855/199544385-48ab727c-7f14-44df-a9bf-a5baaa4e480f.png">
